frontend/src/app/modules/interventions/list/table/table.component.scss

```diff
@@ -1,16 +1,86 @@
 table {
```

Comment:
As we are not using 'table' now, but 'mat-table', would those styles still apply?

Reply:
Yes, it should be `.mat-table` fixed ⚒